eZ/Publish/Core/Persistence/Legacy/Content/UrlAlias/Handler.php

```diff
@@ -427,7 +431,7 @@ protected function createUrlAlias($action, $path, $forward, $languageCode, $alwa
         if ($isPathNew || empty($row)) {
             $data['lang_mask'] = $languageId | (int)$alwaysAvailable;
             $id = $this->gateway->insertRow($data);
-        } elseif ($row['action'] == 'nop:' || $row['is_original'] == 0) {
+        } elseif ($row['action'] === 'nop:' || (int)$row['is_original'] === 0) {
```

Comment:
You are partially do the code refactor so maybe we can introduce const representing 'nop:' action ?

Reply:
> You are partially do the code refactor so maybe we can introduce const representing 'nop:' action ?

Done like that: 1a64d64.